.drone.star

```diff
@@ -249,6 +250,38 @@ def main(ctx):
     pipelineSanityChecks(ctx, pipelines)
     return pipelines
 
+def yarnCache(ctx):
+    return [{
+        "kind": "pipeline",
+        "type": "docker",
+        "name": "cache-yarn",
+        "steps": skipIfUnchanged(ctx, "cache") +
+                 installWebTestRunner() +
+                 rebuildBuildArtifactCache(ctx, "tests-yarn", "/srv/app/web/tests/acceptance/.yarn"),
```

Comment:
This also needs to have a way to access shared volume `stepVolumeOC10Tests`.
I don't know if this step will get access to the shared volume. If not, you might have to change the function or clone web to the src directory directly and build cache from there.

Reply:
other suggestions didn't work. This seems like the way to go